### PR TITLE
[RLlib] SlateQ: Add a hard-task learning test to weekly regression suite.

### PR DESCRIPTION
**Pull request by sven1977**
SlateQ: Add a hard-task learning test to weekly regression suite.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: .buildkite/pipeline.ml.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     - RLLIB_TESTING=1 PYTHON=3.7 ./ci/travis/install-dependencies.sh
     - bazel test --config=ci $(./scripts/bazel_export_options)
       --build_tests_only
-      --test_tag_filters=learning_tests_discrete,-fake_gpus,-torch_only,-tf2_only
+      --test_tag_filters=learning_tests_discrete,-fake_gpus,-torch_only,-tf2_only,-no_tf_static_graph
```
Comment (gjoliver) [inline .buildkite/pipeline.ml.yml:8]: I am just curious about these tags. where can I use them?

**Review by gjoliver — APPROVED** [review]
this PR probably included diffs from the other slateq torch PR.
might want to rebase then merge after the other PR is in.

File: .buildkite/pipeline.ml.yml (hunk 1 above)
Comment (sven1977) [reply .buildkite/pipeline.ml.yml:8]: They are defined in the `rllib/BUILD` file. You can make up any tags and filter by these on the BAZEL command line.

**Comment by sven1977** [comment]
You are right, some diffs need to be merged first. ...

**Pull request merged**